providers/curation/github.js

```diff
@@ -264,9 +262,8 @@ class GitHubCurationService {
     return { name, email, login }
   }
 
-  // return true if patch.skipMultiversionSearch is false and patch has 1 component and 1 revision
-  _isEligibleForMultiversionCuration(patch) {
-    return !patch.skipMultiversionSearch && patch.patches.length == 1 && Object.keys(patch.patches[0].revisions).length == 1
```

Comment:
What's the reason we're removing the skipMultiversionSearch parameter?

Reply:
GitHub event doesn't have `skipMultiversionSearch` property.

Reply:
Gotcha, if we won't ever need it then make sure you remove the parameter from the curation schema. https://github.com/clearlydefined/service/blob/master/schemas/curations-1.0.json